packages/components/src/theme/theme-spectrum/theme-spectrum-overrides.css

```diff
@@ -28,3 +28,40 @@ label[class^='spectrum-'] {
     var(--spectrum-global-dimension-size-85)
   );
 }
+
+[class^='spectrum-Table-headCell'] {
+  /* 
+  Tables are styled with the assumption that the header is the same color as the bg
+  which means they look bad against any other background color, instead just inherit the bg.
+  There is also no variable exposed for this, so we have to target the class directly.
+ */
+  --dh-table-header-cell-background-color: transparent;
+  background: var(--dh-table-header-cell-background-color);
+}
+
+/* Table */
+[class*='spectrum-Table--quiet'] {
+  /* Quiet shouldn't assume a bg color */
+  --spectrum-alias-background-color-default: transparent;
+}
+
+button[class^='spectrum-Button'] {
+  /* make the icon closer to the text */
+  --spectrum-button-primary-text-gap: 6px;
+
+  /* center the text vertically, we have different font, it requires different centering */
```

Comment:
May be worth clarifying that "different font" is with respect to Spectrum's font